libs/core/kiln_ai/datamodel/finetune.py

```diff
@@ -84,13 +85,29 @@ class Finetune(KilnParentedModel):
         default=ChatStrategy.single_turn,
         description="The strategy to use for training the model. 'final_only' will only train on the final response. 'final_and_intermediate' will train on the final response and intermediate outputs (chain of thought or reasoning).",
     )
+    run_config: RunConfigProperties | None = Field(
+        default=None,
+        description="The run configuration for this fine-tune.",
+    )
 
     # Workaround to return typed parent without importing Task
     def parent_task(self) -> Union["Task", None]:
         if self.parent is None or self.parent.__class__.__name__ != "Task":
             return None
         return self.parent  # type: ignore
 
+    def model_id(self) -> str:
```

Comment:
we should rename this. This is a model, it has an`id` field, and it's not this value. model.id and model.model_id are too similar to be different things. 

`nested_id`?